src/DotVVM.Samples.Tests/ErrorsTests.cs

```diff
@@ -83,7 +83,7 @@ public void Error_NotAllowedHardCodedPropertyValue()
                 AssertUI.InnerText(browser.First("[class='exceptionMessage']")
                 ,
                         s =>
-                            s.ToLower().Contains("was not recognized as a valid Boolean.".ToLower())
+                            s.ToLowerInvariant().Contains("was not recognized as a valid Boolean.".ToLowerInvariant())
```

Comment:
```suggestion
                            s.ToLowerInvariant().Contains("was not recognized as a valid boolean.")
```

I'd just prefer this...